app/models/programme_activity.rb

```diff
@@ -18,4 +18,14 @@ class ProgrammeActivity < ApplicationRecord
   belongs_to :activity_type
   belongs_to :activity
   belongs_to :programme
+
+  validate :activity_type_is_also_a_programme_programme_type_activity_type
```

Comment:
Its possible that the list of activity type for a programme type might change. E.g. An admin revises the programme type to add or remove an activity type (perhaps replacing it with a more suitable one).

What will happen in this case as I assume those records will become invalid. Will this cause any problems when saving or updating their programme?